scss/_variables.scss

```diff
@@ -92,20 +94,20 @@ $theme-colors-background: (
 );
 
 $theme-colors-shadow: (
-  primary: theme-color('primary'),
+  primary: theme-color("primary"),
   secondary: $black,
-  success: theme-color('success'),
-  info: theme-color('info'),
-  warning: theme-color('warning'),
-  danger: theme-color('danger'),
-  light: theme-color('light'),
-  dark: theme-color('dark')
+  success: theme-color("success"),
+  info: theme-color("info"),
+  warning: theme-color("warning"),
+  danger: theme-color("danger"),
+  light: theme-color("light"),
+  dark: theme-color("dark")
 );
 
 
 // Options
 $enable-caret:              true !default;
-$enable-rounded:            false;
+$enable-rounded:            true;
```

Comment:
This is not related to lint I guess?

Reply:
Not related to lint at all, and it will run in conflict with another PR opened linked to buttons. New UI Kit style is full of border-radius, we need to activate that to enable border-radius and some lines in some files there needed to user mixin border-radius() instead of plain which needed that variable to be true

Reply:
OK, thanks